src/stores/Frame/FrameStore.ts

```diff
@@ -1536,11 +1800,14 @@ export class FrameStore {
 
             const normalizedNeighbourhood = cursorNeighbourhood.map(pos => AST.normalizeCoordinates(this.wcsInfo, pos.x, pos.y));
 
+            let precisionX = 0;
```

Comment:
`getCursorInfo.precision` is returning zeros. Shall we remove these two variables?